src/main/java/com/bullhornsdk/data/model/entity/core/standard/Note.java

```diff
@@ -20,7 +20,7 @@
 @JsonRootName(value = "data")
 @JsonPropertyOrder({ "id", "action", "bhTimeStamp", "candidates", "clientContacts", "commentingPerson", "comments", "corporateUsers",
 		"dateAdded", "dateLastModified", "entities", "externalID", "isDeleted", "jobOrder", "jobOrders", "jobShifts" , "leads", "migrateGUID", "minutesSpent",
-        "opportunities", "personReference", "placements" })
+        "opportunities", "personReference", "placements", "linkedInID", "people", "primaryDepartmentName", "truestDateAdded" })
```

Comment:
truestDateAdded? The dateAdded isn't that true?

Reply:
Apparently not 😆